api/README.md

```diff
@@ -1,52 +1,73 @@
 
-Run the commands below to initialize the minter account.
+Notable functions:
+- `sendTx()`: Takes a Cadence transaction as an arguement and sends it to write data into the Flow blockchain.
+- `executeScript()`: Takes a Cadence script as an arguement and executes it to read data from the Flow blockchain. 
+- `getLatestBlockHeight()`: Gets the most recent block from the Flow blockchain.
+- `authorizeMinter()`: Returns an asynchronous method to authorize an account to mint Kitty Items.
 
-- **POST /v1/kitty-items/setup** - Create a resource that holds Kitty Items in the `MINTER_FLOW_ADDRESS` account.
+### [Block-cursor Service](src/services/block-cursor.ts)
+This service contains functions that read & write data to the SQL database to maintain the Kitty Items Marketplace
 
-```sh
-curl --request POST \
-  --url http://localhost:3000/v1/kitty-items/setup \
-  --header 'Content-Type: application/json'
-```
+Notable functions:
+- `findOrCreateLatestBlockCursor()`: Queries the `block_cursor` table for the the last block check by the listing handler worker. If no row is found then insert a record with our current latest block height.
+- `updateBlockCursorById()`: Updates the row with the latest block height that has been checked for events.
 
-- **POST /v1/market/setup** - Create a resource that allows the `MINTER_FLOW_ADDRESS` to hold listings for Kitty Items.
+### [Kitty-items Service](src/services/kitty-items.ts)
+This service contains functions that utilize the Flow Service to send/execute transactions/scripts that fetch/manipulate Kitty Items data. These functions are exposed through endpoints to allow users to read & manipulate Kitty Items data on the chain. The Cadence scripts/transactions can also be called in the [Flow-cli](https://docs.onflow.org/flow-cli/) and directly read/write to the Flow blockchain.
 
-```sh
-curl --request POST \
-  --url http://localhost:3000/v1/market/setup \
-  --header 'Content-Type: application/json'
-```
+[Exposed Endpoints](src/routes/kitty-items.ts):
+- **POST `/v1/kitty-items/setup`** *[DEPRECATED]*: Sets up account with a resource that stores Kitty Items in the admin minter account. The [startup script](/.ki-scripts/startup.js) directly sends the setup Cadence transaction instead of using this endpoint. Calls [setup_account.cdc](/cadence/transactions/kittyItems/setup_account.cdc)
 
-## Minting
+- **POST `/v1/kitty-items/mint`** *[DEPRECATED]*: Mint a Kitty Item and send it to a recipient account. Calls [mint_kitty_item.cdc](/cadence/transactions/kittyItems/mint_kitty_item.cdc)
 
-Run the commands below to mint new items and list them for sale.
+- **POST `/v1/kitty-items/mint-and-list`**: Mint a Kitty Item and send it to a recipient account. Create a listing to sell the Kitty Item immediately after. Called by the admin service account to mint & list items onto the store. Calls [mint_and_list_kitty_item.cdc](/cadence/transactions/kittyItems/mint_and_list_kitty_item.cdc)
```

Comment:
```suggestion
- **POST `/v1/kitty-items/mint-and-list`**: Mint a Kitty Item and immediately list it for sale. Called by the admin account to mint and sell new items. Calls [mint_and_list_kitty_item.cdc](/cadence/transactions/kittyItems/mint_and_list_kitty_item.cdc)
```

I would just say "admin account" instead of "admin service account" because "service account" doesn't mean much outside the context of the emulator.